### PR TITLE
Fix cross compilation.

### PR DESCRIPTION
**Pull request by dvc94ch**
this is one of the patches that let's dioxus apps be built for ios with xbuild.

### CODE REVIEW

**Review by wusyong — COMMENTED** [review]
LGTM. Could you add a .change file?

**Comment by dvc94ch** [comment]
Like this? I'm not sure what a .change file is. Added an entry to the change log. If this is ok I'll do the other PR too.

**Comment by dvc94ch** [comment]
Ah ok think I figured it out

File: .changes/crosscompile.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```md
+---
+"wry": patch
+---
+
+Support cross compiling ios on a non macos host.
+
+```
```
Comment (amrbashir) [inline .changes/crosscompile.md]:
```suggestion
---
"wry": patch
---

Support cross compiling ios on a non macos host.

```

Comment (dvc94ch) [reply .changes/crosscompile.md]: :facepalm:

**Review by wusyong — APPROVED** [review]

**Pull request merged**